packages/cli/package.json

```diff
@@ -40,7 +40,7 @@
     "@types/archiver": "5.1.0",
     "@types/micro": "7.3.3",
     "@types/object-hash": "1.3.4",
-    "@types/react": "17.0.0",
+    "@types/react": "17.0.1",
```

Comment:
@patrickhaug do we still need this in cli? I dont think we ever use any of the react types as far as im aware? The react output target is autogen so we never do anything with it?